package.json

```diff
@@ -59,6 +59,9 @@
     "@emoji-mart/react": "^1.1.1",
     "@expo/html-elements": "^0.4.2",
     "@expo/webpack-config": "^19.0.0",
+    "@ffmpeg/core": "^0.12.6",
```

Comment:
do we need this one? looks like it's imported via `unpkg`

Reply:
oh probably not.